subprojects/core-api/src/main/java/org/gradle/process/JavaExecSpec.java

```diff
@@ -76,33 +72,10 @@ public interface JavaExecSpec extends JavaForkOptions, BaseExecSpec {
     @Input
     @ReplacesEagerProperty(
         replacedAccessors = @ReplacedAccessor(value = AccessorType.SETTER, name = "setMain", fluentSetter = true),
-        binaryCompatibility = ACCESSORS_KEPT,
```

Comment:
🤔 Is this still correct?  It's okay to have this migration annotation say it replaces a method "setMain" that no longer exists?

Reply:
Not sure

@asodja @ov7a @lptr what is the correct values for these annotations now?

Reply:
Looking into this further, it seems we have no code that actually reads the value of `replacedAccessors`. This seems more like documentation now than anything.

In that case, I think it would be best that we kept these, stating that this property did indeed replace these getters/setters. 

Looking at the `binaryCompatibility` field, there are two values `ACCESSORS_KEPT` and the default, `ACCESSORS_REMOVED`. 

So there is clearly "room" in this API for documenting accessors that are no longer present. The current state of this PR seems like the correct way to use this